about.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style/about.css">
+</head>
+<body>
+    <header>
+        <ul>
+            <li>
+                <img src="./images/facebook.svg" alt="an image of a facebook icon">
+            </li>
+
+            <li>
+                <img src="./images/twitter.svg" alt="an image of the twitter logo">
+            </li>
+
+            <li>
+                English
+            </li>
+
+            <li>
+                My Page
+            </li>
+
+            <li>
+                Logout
+            </li>
+        </ul>
+
+    <section id="bottom-section">
+        <img src="./images/logo2.svg" alt="an image of the Faris Label logo">
+        <ul>
+            <li class="about">
+                About
+            </li>
+
+            <li>
+                Program
+            </li>
+
+            <li>
+                Join
+            </li>
+
+            <li>
+                Sponsor
+            </li>
+
+            <li>
+                News
+            </li>
+
+            <li>
```

Comment:
- [ ] This tag should be a nav not a section

![dnskndzk](https://user-images.githubusercontent.com/61562626/132832556-87ede991-24d4-42b4-9109-9e078f158416.jpeg)